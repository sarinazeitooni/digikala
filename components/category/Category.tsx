import React from 'react';
import style from './scss/category.module.scss';
import texts from "./texts/texts";
import CategorySwiper from "./category-swiper/CategorySwiper";
import SingleItem from "./single-item/SingleItem";

const Category = () => {
    return (
        <div className={style['category-container']}>
            <div className={style['category-swiper-container']}>
                <div className={style['swiper-header']}>
                    <div className={style['border-dark']}>
                        <h3 className={style['category-title']}>{texts.categorySwiperContainer}</h3>
                        <span className={style['category-subtitle']}>{texts.categorySubTitle}</span>
                    </div>
                    <div className={style['border-light']}/>
                </div>
                <div className={style['swiper-container']}><CategorySwiper/></div>
            </div>
            <div className={style['single-item-advice']}>
                <h3 className={style['single-item-title']}>{texts.singlePicTitle}</h3>
                <span className={style['progress']}></span>
                <SingleItem/>
            </div>
        </div>
    )
}
export default Category;