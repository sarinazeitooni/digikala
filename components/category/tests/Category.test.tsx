import {shallow} from "enzyme";
import Category from "../Category";
describe("category component" , ()=>{
    it("should render correctly" , ()=>{
        const container = shallow(<Category/>);
        expect(container).toMatchSnapshot();
    })
});