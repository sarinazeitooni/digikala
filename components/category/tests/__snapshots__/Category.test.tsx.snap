// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`category component should render correctly 1`] = `ShallowWrapper {}`;
